NEWS.md

```diff
@@ -12,7 +12,8 @@
     - `qs_metabolic`
     - `vs_metabolic`
     - `lb_metabolic`
-    
+
+- OE data was Updated to include pre and post-dose IOP: (#134)   
```

Comment:
```suggestion
- Updated OE dataset to include both pre and post-dose IOP. (#134)   
```

Also would you mind doing me a favour please and updating the item on line 7 ("removing outdated references...") so that it is under a new `## Documentation` section, under the `## New Features`? Then we are good to go.